### DISCUSSION TITLE
Pressure kills too quickly.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13858] [Ideas and suggestions]
When I started the discussion about pressure and the injector headset, I wanted us to have a choice: To wear diving suits or not to wear diving  suits.
Previously, we had no choice, we were forced to wear diving  suits all the time, because the pressure kills us too quickly, the monster attacks are too unexpected, the bots are too stupid.
After the latest update, we again have no choice. 
Now we are forced not to wear diving  suits, since it is too slow to move in them.
But the first three problems have not disappeared.
The game has turned into a pain in the ass.

Maybe I'm wrong, but it seems that it was promised that after updating the game the pressure will not kill as quickly as before.
The main reason for the deaths of bots is that they climb into ballast wearing masks and not in diving suits.
Moreover, even if the pump needs to be repaired, they do not go into the ballast if there are no masks on the sub.
If only the pressure kill not in 5 seconds but in 15 seconds, then the pressure stabilizer inserted into the headset with the injector would have time to work and the bot would remain alive.
Please make the stabilizer work under pressure before the bot dies.

As alternative to give bots skill to apply APS if there is dangerous pressure.
Now bot dies even if he has APS in his inventory.
And we will have the choice again.

**Comment by NotWendy** [comment]
I had a lengthy write-up on this in Discord, but I'll repost my thoughts here, also addressing the parts related to the latest update.

First things first, what the update promised and delivered was a slower flooding rate. Meaning the pressure is still just as deadly as before, but you have more time before it builds up during a flood. Good stuff.

**Thoughts on the current system:**
The main thing I dislike is that **"barotrauma" as an affliction is a binary thing**. Either you have it and you're dead on the spot, or you don't. The only indication that it's building up are the screen effects. The long-term consequences of dipping into a pressurized environment are handled through organ damage, an entirely separate affliction.

The interaction can be made more intuitive if barotrauma itself is allowed to have a range, building up gradually while you're in high pressure, having its own icon, and taking away your health gradually as it builds up.
It'd give players a very clear indication of when they're going to die, and will allow healthier characters to tank more pressure damage before dying.
It'd also allow the pressure stabilizer to be auto-injected into you before you die, as it'd trigger when you're at half health, with 50 points of barotrauma.

Exiting the high pressure environment should allow the affliction to heal on its own, slowly recovering the character's health. An important note is that barotrauma should not count as a regular damage affliction, meaning opiates will do nothing to prolong your life under high pressure.
But maybe liquid oxygenite (used in pressure stabilizers) could provide a slow barotrauma recovery effect. Not enough to survive for long in high pressure by injecting oxygenite repeatedly, but good for recovering your vitality after the flood.

**Notes on PUCS, safety harness, and "barotrauma resistance":**
PUCS was done dirty in the last update. It now slows you down on dry land, which goes against its initial identity as a suit which is particularly good on land, in oxygenated hulls.

PUCS (and maybe safety harness) could provide barotrauma resistance, making the affliction build up slower and prolonging your life in a high pressure environment by a decent amount, but preventing PUCS from acting as a full-time diving suit. That'd keep it useful, but not universally good, and would reinforce its identity as an advanced set of dry land armor.

An alternative idea is to remove the oxygen tank swapping mechanic from PUCS and make it passively refill its supply of oxygen when inside an oxygenated hull. So you're heavily incentivized to use it inside the submarine, but not on long EVA expeditions. But I personally like the idea of barotrauma resistance more. It's a fun buff to the safety harness.
Either way, the speed reduction on dry land needs to go for PUCS specifically.

**Reply by pavel4win** [reply]
Your view of the pressure system is very reasonable, it coincides with mine.
It would be great if the developers listened to us.
Pressure shouldn't kill you in 5 seconds.
Pressure can kill instantly only if a person is directly under the stream gushing from the hole.
Then it's like being hit by a steel ram.
Most of the human body consists of liquid, and if a person is not caught in the stream, he will remain alive, he simply will not be able to breathe, even if he is wearing a mask.

**Comment by pavel4win** [comment]
Regarding the PUCS, I completely agree. It is completely unclear why its speed was reduced.
after all PUCS is abbreviation of
Protective Universal Crisis Suit
here Universal means "on land and in water".
Its main feature is not the ability to take air from outside, but maximum protection from radiation fire and husk.
Thus PUCS indispensable when exploring ruins

Returning to the topic of APS
Bots don't know how to use APS.
if only they can use APS in dangerous pressure then the problem would become less acute.
But they die from pressure even if they have dozen APS in their inventory.

Thus the APS becomes useless.
Bots don’t need APS because they don’t know how to use it.
Players don’t need APS because much more profitable to create oxygenite tanks. 
2 liquid oxygenite = 22 minutes added 30% agility in diving suit.
No one will go outside sub without a diving suit, because the suit gives a bonus to swimming speed and additional protection from wounds.

**Comment by BlackMoons** [comment]
Pressure Stabilizer is already useless because it needs materials you literally can't find anywhere (except ruins?). Iv never once in over 1000 hours of barotrauma ever seen anyone even craft one and I can count on one hand the number of oxygenite shards iv seen harvested.

Buying liquid oxygenite in bulk to survive a single round of bots diving into high pressure for no good reason without a suit is not sustainable, even if vendors sold enough (they don't)

**Reply by pavel4win** [reply]
Each outpost sells at last 4 Liquid Oxygenites (research, mining, town sell 6-7 LO as well)
This is quite enough to provide all bots with APS in 4-5 rounds
The situation when a bot dies from pressure is quite rare, (but unfortunately regular) this happens during an explosive depressurization such as
Kharbda blow
Giant spineling Strike

So APS would be of great benefit to bots.
If only it would work in a headset with an injector, or if they knew how to use it themselves when they get into a compartment with dangerous pressure.
I've played 4000 hours and I still miss this for all 4000 hours.
